api-reference/docstubs/api/user-delta.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "delta"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# delta
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /users/delta
+GET /me/managedDevices/{managedDeviceId}/users/delta
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+If successful, this function returns a `200 OK` response code and a [user](../resources/user.md) collection in the response body.
+
+## Examples
+
```

Comment:
In the live topic, there are examples that show the default response (all the properties specified in the initial query) and the response if a Prefer header is used.

GLOBAL:
In the longer term, just want to know, do we think we will rely on authors to manually provide the appropriate examples?
(I personally think that will be likely.)

Reply:
Angela - I think you're overly fixating on delta as an example - this is a special topic and we've already established that it needs some manual intervention (and probably will even in the longer term).  Can you look at the other topics please, and see if the topic generation looks right for those?